iphone/Classes/TiCalendarCalendar.m

```diff
@@ -174,99 +174,6 @@ - (NSArray *)getEventsBetweenDates:(id)args
   return [TiCalendarEvent convertEvents:events withContext:[self executionContext] calendar:[self calendar] module:module];
 }
 
-- (NSArray *)getEventsInDate:(id)arg
```

Comment:
I don't see this as deprecated in our apidocs, and I don't think this was removed on Android...